test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -75,15 +75,54 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		it( 'Add post tag', async function () {
 			await editorPage.addTag( tag );
 		} );
+
+		afterAll( async function () {
+			// For mobile, but doesn't hurt to do this for Desktop either.
+			await editorPage.closeSettings();
+		} );
 	} );
 
-	describe( 'Preview', function () {
-		let previewPage: Page;
+	describe( 'Jetpack features', function () {
+		it( 'Open Jetpack settings', async function () {
+			await editorPage.openSettings( 'Jetpack' );
+		} );
+
+		skipItIf( envVariables.TEST_ON_ATOMIC !== true )(
+			'Enter SEO title and preview',
+			async function () {
+				await editorPage.enterSEODetails( {
+					title: 'SEO example title',
+					description: 'SEO example description',
+				} );
+			}
+		);
+
+		it( 'Open social preview', async function () {
+			await editorPage.expandSection( 'Social Previews' );
+			await editorPage.clickSidebarButton( 'Open Social Previews' );
+		} );
 
-		// This step is required on mobile, but doesn't hurt anything on desktop, so avoiding conditional.
-		it( 'Close settings sidebar', async function () {
+		it( 'Show social preview for Tumblr', async function () {
+			// Action implemented as "raw" calls for now (2023-09).
+			const editorParent = await editorPage.getEditorParent();
+			const dialog = editorParent.getByRole( 'dialog' );
+
+			await dialog.getByRole( 'tab', { name: 'Tumblr' } ).click();
+			await dialog.getByRole( 'tabpanel', { name: 'Tumblr' } ).waitFor();
+		} );
+
+		it( 'Dismiss social preview', async function () {
+			await page.keyboard.press( 'Escape' );
+		} );
+
+		afterAll( async function () {
```

Comment:
I've decided to remove the step `Close settings sidebar` and replace it with an `afterAll` hook. Closing the panel is not really a valid test on its own, so having it count as a step felt unnecessary.